openjdk.test.modularity/src/tests/com.test.jlink/native/makefile

```diff
@@ -132,11 +132,11 @@ ifeq ($(PLATFORM),win_x86-64)
 endif
  
 ifeq ($(PLATFORM),bsd_x86-64)
-	CC=cc
-	LD=cc
-	IFLAGS=-I. -I$(HEADERDIR) -I$(JAVA_HOME)/include/$(shell uname | tr "[:upper:]" "[:lower:]") -I$(JAVA_HOME)/include -I/usr/include
-	CFLAGS=-D_JNI_IMPLEMENTATION_ -D_TRIVIAL_AGENT -O0 -g3 -pedantic -c -Wall -std=c99 -fPIC -fno-omit-frame-pointer -m64 -o $(OBJDIR)/$(SRC)$(OSUFFIX)
-	LFLAGS=-shared -m64 -o
+    CC=cc
```

Comment:
Do we need these indentation changes?  The only real change in this PR seems to be the addition of `-dynamiclib` in LFLAGS. If we don't need the rest of the indentation changes, we probably should remove them from this PR for sake of consistency.

Reply:
Those "indentation changes" are replacing tabs which should only be used at the start of target commands with spaces.

Reply:
I see. Just making sure it's a valid change.